### PR TITLE
Update migrations to include uatom IBC denom money market

### PR DESCRIPTION
**Pull request by denalimarsh**
Update hard and committee module migrations to include uatom's IBC denom. I set the initial limit is set to ~$1m.

### CODE REVIEW

**Comment by denalimarsh** [comment]
I also made the assumption that the pricefeed would be `atom:usd:30`.

**Review by rhuairahrighairidh — CHANGES_REQUESTED** [review]
Hard params look good. I think the hard governance committee will also need the new permission. Otherwise the committee won't be able to vote through changes to hard perissions. @karzak to confirm.

**Comment by karzak** [comment]
@rhuairahrighairidh is correct- the hard committee needs Atom money market permission added. It also needs swp money market permission (forgot to do this in an early pr).

**Comment by denalimarsh** [comment]
Added `swp` and the uatom ibc denom to the Hard Governance committee. My solution targets committee by ID, so I updated the ID of `Kava God Committee (testing only)` so the Hard Governance committee ID matches up with its mainnet ID.

File: x/committee/legacy/v0_16/migrate.go [hunk 1]
```diff
@@ -107,7 +108,7 @@ func migrateSubParamPermissions(permission v015committee.SubParamChangePermissio
 		}
 
 		// add new requirement for stability committee
-		if isStabilityCommittee {
+		if committeeID == 1 {
```
Comment (DracoLi) [inline x/committee/legacy/v0_16/migrate.go:111]: thoughts on creating some vars in this file for these magic numbers so it's clear the targeting committees?

Comment (denalimarsh) [reply x/committee/legacy/v0_16/migrate.go]: Added in https://github.com/Kava-Labs/kava/pull/1132/commits/6f1b340758b6c52d4d5de874abe6d6812c141522

**Review by DracoLi — APPROVED** [review]
LGTM

**Review by karzak — APPROVED** [review]
LGTM - manually ran mainnet migrations and confirmed that Hard + Committee state looks good.

**Review by rhuairahrighairidh — APPROVED** [review]
change look good, I'll fix the merge conflicts

**Pull request merged**